sdwan_profile_parcels.tf

```diff
@@ -0,0 +1,11 @@
+resource "sdwan_cli_config_profile_parcel" "cli_config_profile_parcel" {
+  for_each = {
+    for cli in try(local.feature_profiles.cli_profiles, {}) :
+    "${cli.name}-config" => cli
+    if lookup(cli, "config", null) != null
```

Comment:
We typically use try() instead of lookup() here:

```
if try(cli.config, null) != null
```

Reply:
done in https://github.com/netascode/terraform-sdwan-nac-sdwan/pull/28/commits/294cb9c0b01bb82cdd6359f0e163ae736a2a8abf